package/MDAnalysis/analysis/base.py

```diff
@@ -126,9 +126,14 @@ def _setup_frames(self, trajectory, start=None, stop=None, step=None):
         """
         self._trajectory = trajectory
         start, stop, step = trajectory.check_slice_indices(start, stop, step)
-        self.start = start
-        self.stop = stop
-        self.step = step
+        if start is not None:
```

Comment:
Quick question here, is there a case where `check_slice_indices` returns `None` for either of `start`, `stop` or `step`? As far as I was aware it already returned ints?

Reply:
I am sorry got confused with issue #2206 requirements asking to Adjust AnalysisBase.__init__() so that start/stop/step are stored in the respective private variables. Also make sure that the DeprecationWarning is raised even if any of the deprecated kwargs is None (currently missing).

Will remove that part for now and work on it after solving this one